x/evm/keeper/state_transition.go

```diff
@@ -519,15 +519,21 @@ func (k *Keeper) ResetGasMeterAndConsumeGas(ctx sdk.Context, gasUsed uint64) {
 	ctx.GasMeter().ConsumeGas(gasUsed, "apply evm transaction")
 }
 
+func getProposerAddress(ctx sdk.Context, proposerAddress []byte) []byte {
```

Comment:
nit: Would be great to add function doc for consistency

Reply:
Added, see if it's ok.